src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/Cells/DiaryOverviewDayCellModel.swift

```diff
@@ -94,9 +98,17 @@ final class DiaryOverviewDayCellModel {
 	var formattedDate: String {
 		diaryDay.formattedDate
 	}
+	
+	func computeExposureHistoryDetail() -> String {
+		if minimumDistinctEncountersWithHighRisk > 0 {
```

Comment:
`> 0` is problematic if you return -1 for non-existent days. I guess it's better to check for `== 0` and return the `.riskTextLowRiskEncountersCause` then. I changed it already in the suggestion above.

Reply:
This is on purpose i.e., -1 for non existent days. 

- UI element _High Risk due to Low Risk Encounters_ - displayed if `Risk Level by Date` for the date associated with the day of the list item is `High` and `Minimum Distinct Encounters With High Risk per Date` for that day is `0` or does not have a value. (-1 in our case)

Reply:
Okay, that seems to be correctly implemented according to the tech spec then 👌🏼